_posts/2020-02-24-release-2.1.0.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Release v2.1.0
+tags: ["v2.1.0", "changelog"]
+excerpt_separator: <!--more-->
+---
+
+As of 24/02 2020 the new version 2.1.0 is out
```

Comment:
Did we release today or on Friday?

Reply:
👍  Good point!. We published artifacts on Friday but didn't announce it yet. So not sure which date should be the release date. Probably Friday, right?

Reply:
I would go for Friday. Or at least that was the intention at the time of writing it.